examples/xml-json-same-objects/sample/sample.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
The sample.json document especially is so much easier to read than the XML version!

Reply:
Glad to hear so 😃